src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -0,0 +1,227 @@
+        <Link
+          href="https://www.nypl.org/help/about-nypl/legal-notices/privacy-policy"
+          type="external" // @todo The external link icon is slightly smaller in the Figma than the default served up by the Link component. I am unsure if/how to manipulate it.
+          id={"privacy"}
+        >
+          Privacy Policy
+        </Link>
+      );
+
+      return (
+        <Stack
+          direction={isLargerThanMobile ? "row" : "column"}
+          ref={ref}
+          __css={styles}
+          {...rest}
+        >
+          {/* Open info div */}
+          <Stack direction={isLargerThanMobile ? "row" : "column"} id={"info"}>
+            <Box
+              bg={getSectionColors(newsletterSignupType, "primary")}
+              id={"color-box"}
+              width={isLargerThanMobile ? ".75rem" : null} // @todo Not sure how to adapt this into the theme.ts and have it work the same.
+              height={isLargerThanMobile ? null : ".75rem"} // @todo ibid.
+            >
+              &nbsp;
+            </Box>
+            <VStack id={"pitch"}>
+              <Heading level={"h3"} text={title} />
+              <Text>{descriptionText}</Text>
+              {privacyPolicy}
+            </VStack>
+          </Stack>
+          {/* Close info div */}
+          {/* Begin action div */}
+          <VStack id={"action"}>
+            {/* Initial form Screen */}
+            {formView && (
+              <>
+                <Form id="newsletter-form" onSubmit={internalOnSubmit}>
+                  <FormField key={"formfield-input"}>
+                    <TextInput
+                      id={"email-input"}
+                      isRequired
+                      invalidText="Please enter a valid email address." // @todo This could be set to helperText value when `if (isInvalid === true)`, allowing devs to set text displayed when bad email.
+                      isInvalid={isInvalidEmail}
+                      labelText="Email Address"
+                      helperText={formHelperText}
+                      name={"email"}
+                      onChange={(e) => setEmail(e.target.value)} // e.target.value is what the user has input. So when they hit "submit" it will be stored in whatever variable we wish in the setEmail function.
+                      placeholder="Enter your email address here"
+                      type="email"
+                      value={email}
+                    />
+                  </FormField>
+                  <FormField key={"formfield-button"}>
+                    <Button
+                      id="submit"
+                      isDisabled={buttonClicked}
+                      key="submit"
+                      type="submit"
+                    >
+                      Submit
+                    </Button>
+                  </FormField>
+                </Form>
+              </>
+            )}
+
+            {/* Confirmation Screen */}
+            {confirmationView && (
+              <>
```

Comment:
You technically don't need the react fragment element `<>` here because the Box is a single top level element.

Reply:
This is fixed